src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -157,47 +166,49 @@ - (void) setLearningRate:(float)lr {
   [_adamBeta setLearningRate:lr];
 }
 
-- (void) updateIndex:(NSUInteger)style {
-  _currentStyle = style;
+- (NSUInteger) styleIndex {
+  return _styleIndex;
+}
+
+- (void) setStyleIndex:(NSUInteger)styleIndex {
```

Comment:
Objective C should auto-synthesize "trivial" getters and setters like this (if you simply omit the declarations of the methods, just include the `@property` declaration)